www/scipylib/building/windows.rst

```diff
@@ -32,6 +32,203 @@ information than this document will provide.
 .. _OpenBLAS: https://github.com/xianyi/OpenBLAS
 .. _LAPACK: http://www.netlib.org/lapack/
 
+
+Building the Released SciPy
+---------------------------
+
+This section provides the step-by-step process to build the released scipy. If you want
+to build completely from source, you should estimate at least three hours to build all
+libraries and compile SciPy. Feel free to stop and inspect any step at any time, but
+for this section, we'll just mention the steps without providing an in-depth explanation
+for the reasons behind them. If you have further questions about what we're doing, more
+in-depth documentation is provided in the sections below. Also, please make sure to read
+this section before proceeding, as the presence or absence of error messages in general
+is not a good indication of whether you've completed a step correctly. Each step creates
+particular files, and what ultimately matters is whether you have built the required files
+rather than whether error messages appeared in your terminal.
+
+First, we need to install the software required to build OpenBLAS_, which is the BLAS_
+library that we're going to use. Because the software to build OpenBLAS is different than
+that required to build SciPy and because OpenBLAS takes a long time to build, we're going
+to start building OpenBLAS first and then explain what to do next while the OpenBLAS build
+is running.
+
+First, install MSYS2 using `these instructions`_. Make sure to install the correct
+architecture for the SciPy that you want to build (eg. 32 or 64 bit), and make sure to
+install the MinGW, and git packages. Open a MSYS2 terminal and type the following:
+
+.. code:: shell
+
+   pacman -S --needed base-devel 
+       mingw-w64-i686-toolchain
+       mingw-w64-x86_64-toolchain git
+       mingw-w64-i686-cmake mingw-w64-x86_64-cmake
+
+You should now have all of the tools required to build OpenBLAS. Now change directories
+to your favorite location with the following:
+ 
+.. code:: shell
+
+   cd /c/Users/MyUser/Downloads/
+   
+You don't necessarily need to build in that particular location, but it should be somewhere
+reasonable. Now clone the repository required to build OpenBLAS:
+
+.. code:: shell
+
+   git clone https://github.com/matthew-brett/build-openblas.git
+   cd build-openblas
+   git submodule update --init --recursive
+
+Now let's build OpenBLAS. To make sure that we're ready to build, type the following in
```

Comment:
Repeating my comment before - there will be some people here (maybe most) who don't want to build OpenBLAS - how about pointing them to the built binary / appveyor repo, and putting these instructions in another page?

Reply:
Okay, but the ATLAS builds won't work (I think you may have suggested them).

Reply:
No, I meant the binary builds of OpenBLAS.